spring-integration-file/src/main/java/org/springframework/integration/file/config/FileTailInboundChannelAdapterFactoryBean.java

```diff
@@ -77,6 +80,15 @@ public void setNativeOptions(String nativeOptions) {
 		}
 	}
 
+	/**
+	 * If false, thread for capturing stderr will not be started
+	 * and stderr output will be ignored
+	 * @param enableStatusReader true or false
```

Comment:
Add `@since 5.0`.
Although if you decided with @garyrussell to backport the fix to `4.3.x`, the version must be `4.3.6`